File-Formats/DocIO/Working-with-Paragraph.md

```diff
@@ -1765,6 +1765,84 @@ By executing the above code example, it generates output Word document as follow
 
 ![Output of Word document with Image caption](WorkingWithImages_images/ImageCaption.png)
 
+### Add SVG image
+
+You can append SVG image specified by the byte array to the end of a paragraph using [AppendPicture](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.IWParagraph.html#Syncfusion_DocIO_DLS_IWParagraph_AppendPicture_System_Byte___System_Byte___) API.
```

Comment:
To add an SVG image to a paragraph in a Word document using Syncfusion DocIO, you can use the AppendPicture() API. 

Note: To preserve the SVG image in the Word document, you need to pass both the SVG image data and the equivalent bitmap image bytes to DocIO.

Reply:
Modified